core/src/main/java/io/ran/Clazz.java

```diff
@@ -493,4 +493,16 @@ public Type getOwnerType() {
 		}
 		return null;
 	}
+
+	public boolean is(Clazz<?> clazz) {
+		return this.clazz.equals(clazz);
```

Comment:
Minor: the member field is not in the beginning of the class, which was odd
Medium: how will this ever return something other than false? Should it not be:
```suggestion
		return this.clazz.equals(clazz.clazz);
```
Medium: Since classes can inherit from each other, `is(...)` could mean equals, it could mean can-an-object-of-this-type-be-assigned-to-variable-of-that-kind. Maybe a javacomment to make it clear what is being tested

Reply:
34e05f1